### ISSUE TITLE
For discussion: fake 'A/B' test with identical variations (a=b) returns B outperforming A by +28%, and with 99% confidence !

### ISSUE THREAD

**Issue by feliperaul** [splitrb/split#199]
I created fake a/b that has literally no code inside it. It just runs on my landing page to trigger Split tracking, and I finish the experiment when someone signs up. 

The code is literally just this:
```
<% ab_test "Pricing_page_conversion", "a", "b" do |style| %>
<% end %>
```

So, A=B.

Imagine my surprise when I saw this today on my split dashboard:

![image](https://f.cloud.github.com/assets/743879/939761/95e7869a-011c-11e3-9c2f-e52ec6dd3c98.png)

I'm in love with Split and I think it's one of the most important gems we use, but I'm posting this so that we can discuss a little bit about the math of the test so that we can understand how a 99% confidence can emerge, with such low numbers, when A=B. 

I decided to take this numbers to a couple of A/B test calculators easily found on google (first results). 

On this site, http://www.usereffect.com/split-test-calculator, it tells me I haven't even reached 90% significance, and I need 131 more visitors to do so. 

```
Sorry, you have no clear winner
We estimate that you'll need 131 more visitors*

Group A:  6.04% conversion
Group B:  2.76% conversion

 	 Visitors:	 Goals:
 Group A: 		
 Group B: 		

*Based on the current trend of your results, your test would require 131 additional visitors to reach significance at the 90% confidence level. Please note that this is at best a loose estimate, designed to help you decide whether or not to continue the test.
````

On this other site (http://www.abtester.com/calculator/), it tells me I have only 91.61% confidence with these numbers.

![image](https://f.cloud.github.com/assets/743879/939812/efada104-011d-11e3-8d5c-f011b91f84ed.png)

Even if I multiply them by two, I don't reach 99% confidence yet:

![image](https://f.cloud.github.com/assets/743879/939822/12c070ae-011e-11e3-8143-460789707e26.png)

Last, I ran it on Visual Website Optimezer calculator, and here are the results (no significance as well):

![image](https://f.cloud.github.com/assets/743879/939856/c479847a-011e-11e3-8034-020b3c319833.png)

I'd love to hear your thoughts.

**Comment by feliperaul** [comment]
Decided to give it another try.

This is the code

`<% ab_test('statistical_validity','A','B') do |block| %>
          <% end %>`

So `A == B`

This is the result. 95% confidence that one is better then the other =/

![image](https://cloud.githubusercontent.com/assets/743879/14143528/217d8d20-f662-11e5-8b6b-0312e05d2186.png)

**Comment by nfm** [comment]
@feliperaul I punched those numbers into Thumbtack's split test calculator and got the same result: https://www.thumbtack.com/labs/abba/#Baseline=11204%2C88818&Variation+1=11568%2C89193&abba%3AintervalConfidenceLevel=0.95&abba%3AuseMultipleTestCorrection=true

It calculates the p-value as 0.025, which is > 95% confidence. I haven't checked this against any other calculators, but I assume they would return similar results.

At 95% confidence, 5% of your tests will see a false positive result - I think this test probably just falls into that category.

I think your other comments about how Split handles small sample sizes might have been addressed by #221, but it's possible there are still issues to tackle there.
